conda/meta.yaml

```diff
@@ -9,8 +9,6 @@ source:
 build:
   noarch: generic
   script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
-  entry_points:
-    - run_ess_csc = lsst.ts.ess.csc.cli:execute_csc
```

Comment:
Why were you able to get rid of this? My CSCs have an entry like this (though not one that uses a "cli module) and I thought it was considered necessary?

Reply:
I thought it was considered necessary as well but Tiago pointed out (a few months ago actually) that having the entry point in pyproject.toml is enough. I tested it locally and it indeed works without having this in the conda recipe as well.

Reply:
Great. Thanks!